File: VotingApp_Client_WEB/Models/VoterEgress.js
// Define a class for user data outgoing
export class VoterEgress {
    constructor() {
        this._name = null;
        this._password = null;
    }

    // Getter and setter for the 'name' property
    get Name() {
        return this._name;
    }
    set Name(value) {
        this._name = value;
    }

    // Getter and setter for the 'password' property
    get Password() {
        return this._password;
    }
    set Password(value) {
        this._password = value;
    }

    // Method to convert the object to JSON with custom property names
    toJSON() {
        return {
            name: this._name,
            password: this._password
        };
    }
}

File: VotingApp_Client_WPF_non_functional/VoterEgress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace VotingApp_Client_WPF
{ 
    // container class for user data outgoing 
    public class VoterEgress
    {
        private string? _name;
        private string? _password;

        public VoterEgress() { }

        [JsonPropertyName("name")]
        public string? Name { get { return _name; } set { _name = value; } }

        [JsonPropertyName("password")]
        public string? Password { get { return _password; } set { _password = value; } }
    }
}
